picky-asn1-x509/src/algorithm_identifier.rs

```diff
@@ -306,7 +306,7 @@ pub enum AlgorithmIdentifierParameters {
     None,
     Null,
     Aes(AesParameters),
-    Ec(EcParameters),
+    Ec(Option<EcParameters>),
```

Comment:
This field should be wrapped in Option as it's said that `parameters` of `AlgorithmIdentifier`'s is optional [Elliptic Curve Cryptography Subject Public Key Information](https://datatracker.ietf.org/doc/html/rfc5480#section-2).

Reply:
Did you consider using `AlgorithmIdentifierParameters::None` value?